### PR TITLE
f-checkout@v0.110.0 - Display correct error message when mobile number field is empty

### PR DESCRIPTION
**Pull request by 1itvinka**
### Changed
- Display 'Please enter your phone number' when the mobile number field is empty, and invalid phone message when the field is invalid but not empty

### CODE REVIEW

**Comment by siggerzz** [comment]
@1itvinka Is it possible to add a visual regression test for the invalid phone number error?

I'm happy to pair on it with you if needed 👍🏼

**Review by dandel10n — DISMISSED** [review]
I have no experience with percy tests but everything else looks good. A pr needs an approve from one of our testers as well

**Review by JoshuaNg2332 — DISMISSED** [review]

File: packages/components/organisms/f-checkout/test/component/f-checkout-delivery.component.desktop.spec.js [hunk 1]
```diff
@@ -29,4 +29,32 @@ describe('f-checkout "delivery" component tests', () => {
         // Assert
         expect(checkout.isPostcodeTypeErrorDisplayed()).toBe(false);
     });
+
```
Comment (siggerzz) [inline packages/components/organisms/f-checkout/test/component/f-checkout-delivery.component.desktop.spec.js]: These tests can be deleted as we've got the visual regression tests

File: packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js [hunk 2]
```diff
@@ -70,7 +70,8 @@ module.exports = class Checkout extends Page {
         },
         mobileNumber: {
             get input () { return $(FIELDS.mobileNumber.input); },
-            get error () { return $(FIELDS.mobileNumber.error); }
+            get error () { return $(FIELDS.mobileNumber.error); },
```
Comment (siggerzz) [inline packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js]: Change to `get emptyError () { return $(FIELDS.mobileNumber.emptyError); },`

**Review by siggerzz — CHANGES_REQUESTED** [review]
A few minor comments, but looks good!

**Review by dandel10n — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

**Pull request merged**